### PR TITLE
Update list.template.jade

### PR DESCRIPTION
**Pull request by ahsang**

### CODE REVIEW

**Comment by ahsang** [comment]
@dominiksteiner @amirmehmood please review and merge

**Comment by ahsang** [comment]
this is now ready to be merged,
addresses https://github.com/BroadsoftLabs/hubCore/issues/771

**Comment by dominiksteiner** [comment]
lgtm

File: app/directives/list/list.template.jade [hunk 1]
```diff
@@ -1,14 +1,15 @@
 span.list
-	span.value(ng-click="visible = !visible")
-		img.icon(ng-if="value && value.icon || value && value.iconClass", src='{{value.icon}}', class='icon-{{value.iconClass}}')
-		a.label(ng-if="value && value.initials || value.label", ng-bind-html='value.initials || value.label', title="{{title}}")
-	.list-items-popup(ng-show='visible')
-		.searchHolder(ng-show="searchVisible")
+	span.value(ng-click="handleTrigger()")
+		img.icon(ng-if="model.value && model.value.icon", src='{{model.value.icon}}')
+		i.icon(ng-if="model.value && model.value.iconClass || model.iconClass",class='icon-{{model.value.iconClass || model.iconClass}}' ng-class="model.value.iconClass ? 'savedSuccess' : ''")
```
Comment (ahsang) [inline app/directives/list/list.template.jade]: @dominiksteiner @amirmehmood added an ng-class here and removed successColor from gMail

File: app/styles/list.styl [hunk 2]
```diff
@@ -1,4 +1,6 @@
 @import 'colors'
+.savedSuccess
```
Comment (dominiksteiner) [inline app/styles/list.styl]: move this under the .list element at least

**Comment by dominiksteiner** [comment]
lgtm

**Pull request merged**